sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/ReadFromKafkaDoFn.java

```diff
@@ -511,6 +532,9 @@ public ProcessContinuation processElement(
             }
           }
         }
+        LOG.warn(
+            "{} records were skipped because of seek returning an earlier position.",
```

Comment:
how about
{} records were skipped due to seek returning to earlier position than requested position of {}

Reply:
Done.